src/client/fuse_ll.cc

```diff
@@ -1349,6 +1384,10 @@ int CephFuse::Handle::init(int argc, const char *argv[])
 
 int CephFuse::Handle::start()
 {
+  if (already_mounted(mountpoint)) {
+    return EALREADY;
```

Comment:
When the `already_mounted()` fails to open the filesystem description file `/etc/mtab` should we return directly and abort the mounting from here. IMO it makes no sense to continue.

Reply:
Done